tests/text_processing/tickets_data/PHPBB3-10922.txt

```diff
@@ -1 +1,5 @@
-[email]user@example.org[/email][email=user@example.org]...[/email]
\ No newline at end of file
+[email]user@example.org[/email]
+[email=user@example.org]...[/email]
```

Comment:
Ah ok. I didn't see it because it was a typo. I added both cases although they are not specific to `[email]`. They should probably be in tests/text_formatter/s9e/default_formatting_test.php rather than this ticket's regression test.